docs/source/release_notes.rst

```diff
@@ -6,84 +6,92 @@
-- ``zos_apf`` - When trying to remove a library that contains the '$' character in the name from APF(authorized program facility), operation will fail.
-- In the past, choices could be defined in either lower or upper case. Now, only the case that is identified in the docs can be set, this is so that the collection can continue to maintain certified status.
+   - Added support for fetching generation data groups (GDG) and generation data sets (GDS).
+   - Added support for data set names with special characters ($, /#, /- and @).
 
+- ``zos_find``
 
-Version 1.9.2
-=============
+   - Added support for finding generation data groups (GDG) and generation data sets (GDS).
+   - Added support for data set names with special characters ($, /#, /- and @).
+
+- ``zos_job_submit``
+
+   - Improved the mechanism for copying to remote systems by removing the use of deepcopy, which had previously resulted in the module failing on some systems.
+   - Added support for running JCL stored in generation data groups (GDG) and generation data sets (GDS).
+   - Added support for data set names with special characters ($, /#, /- and @).
+
+- ``zos_lineinfile``
+
+   - Added support for GDG and GDS relative name notation to specify the target data set and to backup into new generations.
+   - Added support for data set names with special characters ($, /#, /- and @).
+
+- ``zos_mount`` - Added support for data set names with special characters ($, /#, /- and @).
+- ``zos_mvs_raw``
+
+   - Added support for GDG and GDS relative name notation to specify data set names.
+   - Added support for data set names with special characters ($, /#, /- and @).
+
+- ``zos_script`` - Improved the mechanism for copying to remote systems by removing the use of deepcopy, which had previously resulted in the module failing on some systems.
+- ``zos_tso_command``
+
+   - Added support for using GDG and GDS relative name notation in running TSO commands.
+   - Added support for data set names with special characters ($, /#, /- and @).
+
+- ``zos_unarchive``
+
+   - Improved the mechanism for copying to remote systems by removing the use of deepcopy, which had previously resulted in the module failing on some systems.
+   - Added support for data set names with special characters ($, /#, /- and @).
 
 Bugfixes
 --------
 
-- ``zos_copy`` - when creating the destination data set, the module would unnecessarily check if a data set is locked by another process. The module no longer performs this check when it creates the data set.
+- ``zos_copy``
+
+   - Fixes the issue that prevents the module from automatically computing member names when copying a file into a PDS/E. The module now computes the member name when copying into a PDS/E.
+   - Fixes an issue that would perform an unnecessary check if a destination data set is locked for data sets the module created. The module now only performs this check for on destinations that are present.
+
```

Comment:
> ...now only performs this check for on destinations that are present.

should remove the "on" (or remove from):
> ..now only performs this check for ~on~ destinations that are present.

Reply:
Done